src/frontend/components/SøknadsSteg/OmBarnaDine/useBarnCheckboxFelt.tsx

```diff
@@ -6,15 +6,25 @@ import { Avhengigheter, feil, Felt, FeltState, ok, useFelt } from '@navikt/famil
 import { useApp } from '../../../context/AppContext';
 import { barnDataKeySpørsmål } from '../../../typer/barn';
 import { BarnetsId } from '../../../typer/common';
+import { FlettefeltVerdier, LocaleRecordBlock } from '../../../typer/sanity/sanity';
 import SpråkTekst from '../../Felleskomponenter/SpråkTekst/SpråkTekst';
 
-const useBarnCheckboxFelt = (
-    datafeltNavn: barnDataKeySpørsmål,
-    feilmeldingSpråkId: string,
-    avhengighet: Felt<ESvar | null>,
-    avhengigJaNeiSpmSvarCondition = ESvar.JA
-) => {
-    const { søknad } = useApp();
+const useBarnCheckboxFelt = ({
+    datafeltNavn,
+    feilmelding,
+    flettefelter,
+    feilmeldingSpråkId,
+    avhengighet,
+    avhengigJaNeiSpmSvarCondition = ESvar.JA,
+}: {
+    datafeltNavn: barnDataKeySpørsmål;
```

Comment:
Kan du legge disse prop-typene som en egen `interface Props` i stedet for å inline de?

Reply:
Enig, det blir bedre å gjøre det slikt :)